app/src/main/kotlin/io/homeassistant/companion/android/notifications/PushManager.kt

```diff
@@ -0,0 +1,21 @@
+package io.homeassistant.companion.android.notifications
+
+import io.homeassistant.companion.android.common.notifications.PushProvider
+import javax.inject.Inject
+
+class PushManager @Inject constructor(
+    val providers: Map<String, @JvmSuppressWildcards PushProvider>
+) {
```

Comment:
Could you create an instrumentation test that checks how the map looks like. It will be useful in the future to make sure the map contains the proper values.